common/lib/capa/capa/templates/jsinput.html

```diff
@@ -52,6 +52,6 @@
   % endif
 
   % if msg:
-      <span class="message">${msg|n}</span>
+      <span class="message" aria-describedby="answer_iframe_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
"answer_iframe_${id}" does not appear anyplace else in the HTML. Is it a legitimate value to be using?